zipline/data/data_portal.py

```diff
@@ -139,6 +139,9 @@ def __init__(self,
         }
         self._equity_daily_reader_array_data = {}
 
+    def _reindex_extra_source(self, df, source_date_index):
```

Comment:
It gets overridden in live_data_portal, which does not forward fill https://github.com/quantopian/qexec/pull/9440